src/EoS/EoS_Init.cpp

```diff
@@ -66,6 +67,10 @@ void EoS_Init()
    EoS_Init_Ptr = EoS_Init_Isothermal;
 #  elif ( EOS == EOS_NUCLEAR )
    EoS_Init_Ptr = EoS_Init_Nuclear;
+#  if   ( NUC_TABLE_MODE == NUC_TABLE_MODE_TEMP )   
+   if ( Flu_ResetByUser_Func_Ptr == NULL ) 
+      Aux_Error( ERROR_INFO, "Flu_ResetByUser_Func_Ptr == NULL for NUC_TABLE_MODE = %d !!\n", NUC_TABLE_MODE_TEMP );
+#  endif
```

Comment:
- We should check `Flu_ResetByUser_API_Ptr`, not `Flu_ResetByUser_Func_Ptr`.
- It is not meaningful to check either `Flu_ResetByUser_API_Ptr` or `Flu_ResetByUser_Func_Ptr` here. If `OPT__RESET_FLUID` is not set, `Flu_ResetByUser_API()` will never be invoked.

Reply:
For the first point, check my general comments. And I agree that we don't need to check anything here. Instead, check `OPT__RESET_FLUID` in `Aux_Check_Parameter()`.

Reply:
I removed this part and added the check `OPT__RESET_FLUID` in `Aux_Check_Parameter()`